certbot-apache/tests/obj_test.py

```diff
@@ -27,14 +27,14 @@ def test_repr(self):
             "certbot_apache._internal.obj.Addr(('127.0.0.1', '443'))")
 
     def test_eq(self):
-        self.assertTrue(self.vhost1b == self.vhost1)
-        self.assertFalse(self.vhost1 == self.vhost2)
+        self.assertEqual(self.vhost1b, self.vhost1)
+        self.assertEqual(self.vhost1, self.vhost2)
```

Comment:
This should be `assertNotEqual`. And I am surprised that the test is passing in this case!